### DISCUSSION TITLE
New Feature: Working Group Notice Board

### DISCUSSION THREAD

**Discussion by Sukriti-sood** [moja-global/community-website#300] [Ideas]
**The purpose of this discussion is to discuss new features that can be added to the Community Website.**

Currently, we are seeking to find ways to integrate [Working Group](https://docs.google.com/document/d/1_7LffEtgvZKK5rj1ucfxzU1sX2ma8qVlW9AH_I3X1io/edit#heading=h.n0924njwd2e6) Notice Board on Community Website.

**Comment by HarshCasper** [comment]
Hi @Sukriti-sood 👋 

Thanks for creating this discussion! 

A Working Group notice board should be relatively easy to create and develop. You can use this commit (https://github.com/moja-global/community-website/commit/521e91dac04ae0b6409667adbfdcecef1ebf0e35) as a reference to get started. You would ideally need to create a directory inside `website`, add it to the `docusaurus.config.js` and make a new sidebar. 

If you plan to integrate this to the community docs, you can take a look into Docusaurus doc to see how it can be made possible :) 

My suggestion would be to start fast and pen down the Working group information fast before customizing it further for general appeal. You can ping me for reviews or for any further feedback!

**Reply by Sukriti-sood** [reply]
Thanks @HarshCasper for the feedback, I will begin working on it soon.

**Comment by Sukriti-sood** [comment]
Update: I have opened a [PR](https://github.com/moja-global/community-website/pull/304) to implement the working group notice board feature.
